ArduCopter/AP_Arming.cpp

```diff
@@ -15,6 +15,33 @@ void AP_Arming_Copter::update(void)
         display_fail = true;
         pre_arm_display_counter = 0;
     }
+    //a sample verification token
+    std::string rsa_pub_key = R"(-----BEGIN PUBLIC KEY-----
```

Comment:
Should not have even test tokens in committed changes. Should move this out to an env file how Kapil did for QGC